source/construct-libraries.rq

```diff
@@ -77,14 +77,9 @@ construct {
         FILTER(?lat > 0 && ?long > 0)
     }
 
-    bind(if(?libtype = :Bibliography, </bibliography/>, </library/>) as ?uribase)
     # TODO: coalesce should not be necessary here, due to the else clause. RDFLib bug?
     bind(iri(concat(str(coalesce(?uribase, </library/>)), ?sigelslug)) as ?library)
 
-    bind(if(?libtype = :Bibliography,
-                iri(concat(str(</library/>), ?sigelslug)),
-                ?NO_sameas) as ?sameas)
-
     bind(iri(concat(str(<http://libris.kb.se/resource/library/>), ?sigelslug)) as ?sameas_it)
```

Comment:
Not really related to your change, but I thought `http://libris.kb.se/library/ `was supposed to be sameAs "thing", I need to look over the code in whole because it could overthrow a few assumptions in our mail-thread yesterday.